Sources/Private/UIViewExtensions.swift

```diff
@@ -1,25 +1,67 @@
 import UIKit
+import XCTest
 
-extension UIView {
-    func renderFormat() -> UIGraphicsImageRendererFormat {
-        let format = UIGraphicsImageRendererFormat(for: .current)
-        format.opaque = true
+extension UITraitCollection {
+    static let snapshots = UITraitCollection(traitsFrom: [
+        UITraitCollection(displayGamut: .P3),
+        UITraitCollection(displayScale: 1.0),
+        UITraitCollection(activeAppearance: .active),
+        UITraitCollection(userInterfaceLevel: .base),
+        UITraitCollection(legibilityWeight: .regular),
+        UITraitCollection(userInterfaceStyle: .light),
+        UITraitCollection(preferredContentSizeCategory: .medium),
+    ])
+}
+
+extension UIGraphicsImageRendererFormat {
+    static let snapshots: UIGraphicsImageRendererFormat = {
+        let format = UIGraphicsImageRendererFormat(for: .snapshots)
+        format.opaque = false
         format.preferredRange = .standard
         return format
-    }
+    }()
+}
+
+extension UIView {
     func renderer() -> UIGraphicsImageRenderer {
-        UIGraphicsImageRenderer(bounds: bounds, format: renderFormat())
+        UIGraphicsImageRenderer(bounds: bounds, format: .snapshots)
     }
     
     func renderLayerAsBitmap() -> UIImage {
-        renderer().image {
-            layer.render(in: $0.cgContext)
-        }
+        renderer().image(actions: renderLayerActions(_:))
+    }
+    
+    func renderLayerAsPNG() -> Data {
+        renderer().pngData(actions: renderLayerActions(_:))
+    }
+    
+    func renderLayerActions(_ context: UIGraphicsImageRendererContext) {
+        configureContext(context)
+        layer.render(in: context.cgContext)
+    }
+    
+    func configureContext(_ context: UIGraphicsImageRendererContext) {
+        context.cgContext.setFlatness(0.01)
```

Comment:
looks like cargo cult: extract configuration that has no effect yet
https://github.com/paulz/ViewSnapshots/projects/1#card-82828085